tests/foreman/ui/test_bookmarks.py

```diff
@@ -249,15 +349,35 @@ def test_negative_update_bookmark_name(self):
         4. Submit
 
         @Assert: Error - name already taken, bookmark not updated
-
-        @Status: Manual
         """
-        for entity in BOOKMARK_ENTITIES:
-            if entity.get('setup'):
-                create_entity(entity['setup'])
-            pass
+        for entity in self.entities:
+            with self.subTest(entity):
+                with Session(self.browser) as session:
+                    bm1_name = gen_random_string()
+                    session.nav.go_to_select_org(self.org_.name)
+                    ui_lib = getattr(self, entity['name'].lower())
+                    ui_lib.create_a_bookmark(
```

Comment:
I'm not sure what are you suggesting here. `getattr(self, entity['name'].lower())` is used to receive and object from its string name. I.e. `getattr(self, entity['name'].lower())` will return `self.activation_key`, `self.product` etc. So i'm receiving lib for current entity and then just calling for example `self.activation_key.create_a_bookmark()`. What do you mean by 'two calls' and 'passing two different names'?